netbeans-l10n/build.xml

```diff
@@ -85,7 +85,7 @@
         Probably you failed to clean your sources before updating. Try: ant hg-clean
     </fail>
     <!-- OK, continue: -->
-    <javac srcdir="antsrc" destdir="build/antclasses" deprecation="true" debug="true" source="1.5" target="1.5">
+    <javac srcdir="antsrc" destdir="build/antclasses" deprecation="true" debug="true" source="1.8" target="1.8">
```

Comment:
The commit message is "Changed netbeans-l10n/build.xml to work with jdk8" which I don't understand. Is that a typo? Should the message be "...to work with jdk11"?

Reply:
Should, of course, be "Changed netbeans-l10n/build.xml to work with jdk8 or higher". 
For whatever reason I couldn't run the script with jdk8 and source/target still set to level 1.5.